src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6314,28 +6315,33 @@ namespace SimAirServingZones {
 				CalcSysSizing( AirLoopNum ).InpDesCoolAirFlow = CalcSysSizing( AirLoopNum ).DesCoolVolFlow * FractionOfAutosize;
 				FinalSysSizing( AirLoopNum ).InpDesCoolAirFlow = FinalSysSizing( AirLoopNum ).DesCoolVolFlow * FractionOfAutosize;
 			} else if ( SELECT_CASE_var == FlowPerCoolingCapacity ) {
-				if ( FinalSysSizing( AirLoopNum ).CoolingCapMethod == FractionOfAutosizedCoolingCapacity ) {
+				if ( FinalSysSizing( AirLoopNum ).CoolingCapMethod == FractionOfAutosizedCoolingCapacity || FinalSysSizing( AirLoopNum ).ScaledCoolingCapacity == DataSizing::AutoSize ) {
```

Comment:
After reviewing the code, the check for ScaledCoolingCapacity == AutoSize is needed because this variable is set to Autosize if the cooling capacity is Autosized. The same is true for heating.